C++20: ambiguous overloads for operator==

When building with Clang 10 in C++20 mode, a lot of comparison operators are now ambiguous.

Example:

    /home/daniel/git/sqlpp11/tests/Interpret.cpp:100:21: error: use of overloaded operator '==' is ambiguous (with operand types 'const sqlpp::column_t<test::TabBar, test::TabBar_::Alpha>' and 'parameter_t<value_type_of<sqlpp::column_t<test::TabBar, test::TabBar_::Alpha> >, sqlpp::column_t<test::TabBar, test::TabBar_::Alpha> >' (aka 'parameter_t<sqlpp::integral, sqlpp::column_t<test::TabBar, test::TabBar_::Alpha> >'))
      serialize(t.alpha == parameter(t.alpha), printer).str();
                ~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~
    /home/daniel/git/sqlpp11/include/sqlpp11/basic_expression_operators.h:127:10: note: candidate function [with T = sqlpp::parameter_t<sqlpp::integral, sqlpp::column_t<test::TabBar, test::TabBar_::Alpha> >]
        auto operator==(T t) const -> _new_binary_expression_t<equal_to_t, T>
             ^
    /home/daniel/git/sqlpp11/include/sqlpp11/basic_expression_operators.h:127:10: note: candidate function (with reversed parameter order) [with T = sqlpp::column_t<test::TabBar, test::TabBar_::Alpha>]

Any ideas how to prevent this / work around this issue?
Thanks!

--- a/include/sqlpp11/column.h
+++ b/include/sqlpp11/column.h
@@ -63,7 +63,8 @@ namespace sqlpp
     template <typename T>
     using _is_valid_assignment_operand = is_valid_assignment_operand<value_type_of<ColumnSpec>, T>;
 
-    // disambiguation for C++20
+    // disambiguation for C++20 / clang
+    // (see https://bugs.llvm.org/show_bug.cgi?id=46508)
     using expression_operators<column_t<Table, ColumnSpec>, value_type_of<ColumnSpec>>::operator==;
     using expression_operators<column_t<Table, ColumnSpec>, value_type_of<ColumnSpec>>::operator!=;
 

--- a/include/sqlpp11/column.h
+++ b/include/sqlpp11/column.h
@@ -63,6 +63,10 @@ namespace sqlpp
     template <typename T>
     using _is_valid_assignment_operand = is_valid_assignment_operand<value_type_of<ColumnSpec>, T>;
 
+    // disambiguation for C++20
+    using expression_operators<column_t<Table, ColumnSpec>, value_type_of<ColumnSpec>>::operator==;
+    using expression_operators<column_t<Table, ColumnSpec>, value_type_of<ColumnSpec>>::operator!=;
+
     column_t() = default;
     column_t(const column_t&) = default;
     column_t(column_t&&) = default;

